includes/payments/functions.php

```diff
@@ -840,7 +840,10 @@ function edd_get_total_earnings() {
 
 				if( ! empty( $payments ) ) {
 					$payments = implode( ',', $payments );
-					$total += $wpdb->get_var( "SELECT SUM(meta_value) FROM $wpdb->postmeta WHERE meta_key = '_edd_payment_total' AND post_id IN({$payments})" );
+
+					foreach( $payments as $payment_id ) {
```

Comment:
In any case, even a single filter for the total would be better than nothing. :smile: